src/card/aggregate-status/aggregate-status-card.html

```diff
@@ -24,3 +24,30 @@ <h2 class="card-pf-title">
     </p>
   </div>
 </div>
+<div ng-if="isMiniLayout" class="card-pf card-pf-aggregate-status card-pf-aggregate-status-mini" ng-class="{'card-pf-accented': shouldShowTopBorder}">
+  <h2 class="card-pf-title">
+    <span ng-if="status.iconClass" class="{{status.iconClass}}"></span>
+    <a ng-if="status.href" href="#{{status.href}}">
+      <span class="card-pf-aggregate-status-count">{{status.count}}</span>
+      {{status.title}}
+    </a>
+    <span ng-if="!status.href">
+      <span class="card-pf-aggregate-status-count">{{status.count}}</span>
+      {{status.title}}
+    </span>
+  </h2>
+  <div class="card-pf-body">
+    <p class="card-pf-aggregate-status-notifications" ng-repeat="notification in status.notifications">
```

Comment:
The repeater here is on the "card-pf-aggregate-status-notifications" rather than on "card-pf-aggregate-status-notification" as it is for the other layouts. 

Currently multiple notifications would stack vertically rather than horizontally. Horizontally would make more sense though I'm not sure we should show multiples for the mini layout anyhow. If not, then it should be prevented.